uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+if c.KIOSK_CC_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.STRIPE, "Pay with credit card now (faster)"])
+
+if c.GROUPS_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.GROUP, "Taking an unassigned Group badge (group leader must be present)"])
```

Comment:
The creation of hashed integer constants for everything under ``[enums]`` is a deliberate feature in how we do config.  It's so central to how we do config that it's one of the few features I actually bothered to write code comments explaining it :)

On that note, any move to a configdb approach is going to need to be able to handle these same use cases.  Whatever it does under the hood is fine so long as we can write code that refers to ``c.ATTENDEE_BADGE`` or ``c.STRIPE``, etc.